app/src/docs/content/dataUnions/dataUnionsCore.mdx

```diff
@@ -24,9 +24,9 @@ To create a Data Union in Core, go to Core > Products, click <span className={do
 ### Deploying a Data Union
 Creating a Data Union starts with deploying the Data Union smart contract. A Data Union is identified by the address of its smart contract. While the deployment transaction is an Ethereum mainnet transaction, the Data Union smart contract is deployed to the sidechain - currently xDai. The sidechain address is deterministic and can be found by reading the mainnet contract on <a href="http://etherscan.io/" target="_blank" rel="noopener noreferrer">Etherscan</a>. Currently the Core UI shows the mainnet address, in the future it will show both the mainnet and sidechain addresses.
 
-The account that deploys the contract is also the owner of that contract. Once all the required fields have been filled and the draft Data Union is ready to be deployed, click Continue to deploy the Data Union. The deploy dialogue will walk you through the deploy process.
+The account that deploys the contract is also the owner of that contract. Once all the required fields have been filled and the draft Data Union is ready to be deployed, click <span className={docsStyles.streamrHighlight}>Continue</span> to deploy the Data Union. The deploy dialogue will walk you through the deploy process.
 
-For the contract deployment (mainnet transaction), you'll need a web3 wallet, and a small amount of Ethereum for gas fees. The Data Union deployment might take a moment to register once the transaction has been mined. The progress indicator will show an estimate of the remaining time. You can either close the dialog and move on or wait for the process to complete after which you will be automatically redirected.
+For the mainnet contract deployment transaction, you'll need a Web3 wallet and a small amount of Ethereum for gas fees. The Data Union deployment might take a moment to register once the transaction has been mined. The progress indicator will show an estimate of the remaining time. You can either close the dialog and move on or wait for the process to complete after which you will be automatically redirected.
```

Comment:
This is good, we could state how much gas (in Gwei) is required as well for DU deployment?

Reply:
If it's know and doesn't fluctuate then why not?